clientconn.go

```diff
@@ -539,12 +540,24 @@ func (cc *ClientConn) WaitForStateChange(ctx context.Context, sourceState connec
 //
 // Experimental
 //
-// Notice: This API is EXPERIMENTAL and may be changed or removed in a
-// later release.
+// Notice: This API is EXPERIMENTAL and may be changed or removed in a later
+// release.
 func (cc *ClientConn) GetState() connectivity.State {
 	return cc.csMgr.getState()
 }
 
+// Connect causes all subchannels in the ClientConn to attempt to connect if
+// the channel is idle.
```

Comment:
Should this marked experimental?
And should we make it more clear in the documentation that the function returns without waiting for all subChannels to actually connect?

Reply:
Yes, it should, thanks.